op-node/metrics/metrics.go

```diff
@@ -261,12 +268,11 @@ func NewMetrics(procName string) *Metrics {
 			Help:      "Histogram of L1 Reorg Depths",
 		}),
 
-		TransactionsSequencedTotal: factory.NewGauge(prometheus.GaugeOpts{
+		TransactionsSequencedTotal: factory.NewCounterVec(prometheus.CounterOpts{
 			Namespace: ns,
 			Name:      "transactions_sequenced_total",
 			Help:      "Count of total transactions sequenced",
-		}),
-
+		}, []string{"type"}),
```

Comment:
Will this break a grafana dashboard? Its ok if it does, just need to loop in the right people to update it

Reply:
Potentially. It would add a couple new timeseries under the same metric which may cause some updates. I doubt this would break alerts.

Reply:
I think it's fine, I'll update our internal dashboards to handle the change